### DISCUSSION TITLE
Crashed while exiting with code 0x80131506. Generating report...

### DISCUSSION THREAD

**Discussion by pedrobagatin** [tsunamods-codes/7th-Heaven#219] [Q&A] [answered]
**Describe the bug**
After 4.0.0 update, was unable to launch the game normally

** Versions **
[ X ] Tested on latest stable version
[ ] Pre-release only bug

**To Reproduce**
1. Open 7th heaven
2. Click "Play" button

**Expected behavior**
Popup appears with the message to use FF7_Launch.exe (using FF7_en.exe), tried to change, but doesn't work.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Version: [e.g. 22] 11
 - Build: [e.g. 19042]

**Additional context**

`00:06:56|INFO|SeventhHeaven.ViewModels.GameLaunchViewModel|### Launching Final Fantasy VII ###
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking System DEP status...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking FF7 is not running ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking FF7 .exe exists at C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY VII\ff7_en.exe ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking FF7 Steam if installed correctly......
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Verifying installed game is compatible ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Creating missing required directories ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Verifying English game files exist ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Verifying music files exist ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Verifying FF7_Launcher exe ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking a profile is active ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking mod compatibility requirements ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking mod constraints for compatibility ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking mod load order requirements ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Copying ff7input.cfg to FF7 path ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|	using control configuration file [Default] Steam KB+PlayStation (Stock).cfg ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|	Copying C:\Users\pedro\AppData\Local\Programs\7th Heaven\Resources\Controls\[Default] Steam KB+PlayStation (Stock).cfg -> C:\Users\pedro\OneDrive\문서\Square Enix\FINAL FANTASY VII Steam\ff7input.cfg ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Creating Runtime Profile ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|	adding paths to monitor ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Copying 7thWrapperLoader.dll to FF7 path (if not found or older version detected) ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Checking if Reunion mod is installed ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|	found: False
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Launching additional programs to run (if any) ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Writing temporary runtime profile file to C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY VII\.7thWrapperProfile ...
00:06:57|INFO|SeventhHeaven.Classes.GameLauncher|Attempting to inject with 7thWrapperLoader ...
00:07:03|INFO|SeventhHeaven.Classes.GameLauncher|Getting FF7 proc ...
00:07:03|INFO|SeventhHeaven.Classes.GameLauncher|Starting programs for mods ...
00:07:03|INFO|SeventhHeaven.Classes.GameLauncher|Starting plugins for mods ...
00:07:03|INFO|SeventhHeaven.Classes.GameLauncher|Setting up FF7 .exe to stop plugins and mod programs after exiting ...
00:07:03|INFO|SeventhHeaven.Classes.GameLauncher|Waiting for FF7 .exe to respond (120 seconds max) ...
00:07:03|INFO|SeventhHeavenUI.ViewModels.MainWindowViewModel|Successfully launched FF7!
00:07:11|INFO|SeventhHeaven.Classes.GameLauncher|Stopping other programs for mods started by 7H ...
00:07:11|ERROR|SeventhHeaven.Classes.GameLauncher|FF7.exe Crashed while exiting with code 0x80131506. Generating report...`

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please make sure you use a clean copy of the Steam game, then uninstall and reinstall 7th and try again.

**Reply by pedrobagatin** [reply]
Hmm.. already tried many times, unfortunately it doesn't work. Reinstalled Steam game and it runs, but after trying to launch via 7th Heaven it doesn't work, even without any mods.

**Reply by julianxhokaxhiu** [reply]
Make sure you install all the dependencies. If you have VC++ dependencies installed, uninstall them all then reinstall 7th INCLUDING the dependencies.

**Reply by pedrobagatin** [reply]
> Make sure you install all the dependencies. If you have VC++ dependencies installed, uninstall them all then reinstall 7th INCLUDING the dependencies.

![image](https://github.com/user-attachments/assets/df9e208a-4b83-47f2-add8-cb6c4c553c22)

same error... when I try to run after install this error appears,

**Reply by pedrobagatin** [reply]
Kind solved the problem by reinstalling v3.5.1.0 version, don't know exactly what the problem is, thankfully saves are still there :)
